### PR TITLE
Add Globus Group management tutorials

### LINKED ISSUE #840: Add group/subgroup management to Globus tutorials
### What would you like to see added?

Add group/subgroup management to Globus tutorials. This should be its own page to avoid crowding other pages. We can cross-link from the subgroup page.

### PR DESCRIPTION
**Pull request by wwarriner**
# Pull Request

## Overview

Add tutorials on the subject of Globus group management.

## Proposed Changes

- Add new page with tutorials
- Update index page
- Rename "Globus for Groups" tutorial to "Globus for Organizations" tutorial to avoid ambiguity on the word "Groups".
- Organized nav section of `mkdocs.yml`.

### Questions addressed

- General information:
    - Where Can I Find the Groups Page in the Globus Web App?
    - How Do I Find a Group?
- For Group members:
    - How Do I Request to Join a Group?
    - How Do I Find and Accept an Invitation From a Group?
    - How Do I Leave a Group?
- For Group administrators:
    - Managing Groups and Subgroups
        - How Do I Create a Group?
        - How Do I Create a Subgroup?
        - How Do I See Subgroups of a Group?
        - How Do I Modify a Group or Subgroup?
            - How Do I Make My Group Invite Only?
            - How Do I Hide My Group from Others?
            - How Do I Keep Members from Seeing Each Other?
        - How Do I Delete a Group or Subgroup?
    - Managing Group Members and Roles
        - How Do I Invite Others to a Group?
        - How Do I See Group Members?
        - How Do I Modify Group Members?
            - How Do I Change a Group Member's Role Level?
            - What Does Each Role Mean?
            - How Do I Remove a Group Member?

## Related Issues

Fixes #840

### CODE REVIEW

**Comment by bdu-birhanu** [comment]
It looks good to me. Below are a few minor fixes.

File: docs/data_management/transfer/tutorial/globus_group_management.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+            - [How Do I Change a Group Member's Role Level?](#how-do-i-change-a-group-members-role-level)
+            - [What Does Each Role Mean?](#what-does-each-role-mean)
+            - [How Do I Remove a Group Member?](#how-do-i-remove-a-group-member)
+
+## Where Can I Find the Groups Page in the Globus Web App?
+
+1. [Get onto the Globus Web App](./globus_individual_tutorial.md#how-do-i-get-onto-the-globus-web-app).
+1. In the left-hand navigation panel, click Groups to be taken to the Groups page. All Group management activities in this tutorial document start on this page, so take some time to familiarize yourself with getting to this point.
+
+    ![Groups page of the Globus Web App.](./images/common/groups.png)
```
Comment (bdu-birhanu) [inline docs/data_management/transfer/tutorial/globus_group_management.md:39]: I would suggest including the "Groups" navigation tab in the screenshot.

File: docs/data_management/transfer/tutorial/globus_group_management.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+
+## How Do I Find a Group?
+
+1. [Get to the Groups page of the Globus Web App](#where-can-i-find-the-groups-page-in-the-globus-web-app).
+1. If you are looking for new groups, ensure the "MY GROUPS" Checkbox is unchecked. If you are looking for groups you are already a member of, ensure the "MY GROUPS" Checkbox is checked.
+
+    ![My groups checkbox unchecked.](./images/common/groups-checked.png)
```
Comment (bdu-birhanu) [inline docs/data_management/transfer/tutorial/globus_group_management.md:46]: Would be good if you could  mark or add an arrow pointing to the 'My Groups' button in the screenshot.

File: docs/data_management/transfer/tutorial/globus_group_management.md [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+    1. **Terms & Conditions** (optional): should not be necessary for UAB groups.
+    1. **Enable High Assurance** checkbox: Check if your group will deal with PHI, HIPAA, or other protected data.
+    1. **Users may request to join this group** checkbox: Uncheck to make the group invite-only. Core administrators should consider leaving this box checked for their top-level group to avoid work filtering invitations and reduce noise.
+    1. **Signup Fields** buttons: First Name, Last Name, and Organization are always required. We recommend also selecting "Project", "Department", and "Field of Science". Feel free to select any fields you wish.
+    1. **Group Visibility** radio buttons: Click "Members of this Group" to hide the group from non-members.
+    1. **Membership Visibility** radio buttons: Click "Administrators and Managers Only" to hide member names from each other. Core adminstrators should consider using the "Administrators and Managers Only" option for their top-level groups because it improves the user experience when members are looking for administrator names.
+
+    ![Create Group form.](./images/gg-create-group/002-create-group-form.png)
```
Comment (bdu-birhanu) [inline docs/data_management/transfer/tutorial/globus_group_management.md:129]: The image is labeled with numbers, but the text descriptions above uses letters. Is it possible to use either numbers or letters consistently?

File: docs/data_management/transfer/tutorial/globus_group_management.md [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+## How Do I Remove a Group Member?
+
+1. [Prepare to modify Group memberships](#how-do-i-modify-group-members).
+1. Click the "Remove Membership" button to immediately remove the member from the group. There is no confirmation for this action.
+
+    ![Membership page with highlighted remove member button.](./images/gg-modify-members-remove/002-membership-page.png)
+
+1. The page should update to reflect the change immediately. Instead of the "Remove Membership" button, you should see "Add Membership" and "Send Invitation" buttons.
+
+    ![Membership page with highlighted remove member button.](./images/gg-modify-members-remove/003-membership-page-after.png)
```
Comment (bdu-birhanu) [inline docs/data_management/transfer/tutorial/globus_group_management.md:273]: Add arrow pointer or mark the two buttons in the screenshot.

File: docs/data_management/transfer/tutorial/globus_group_management.md (hunk 2 above)
Comment (wwarriner) [reply docs/data_management/transfer/tutorial/globus_group_management.md:46]: Go this one and the unchecked version.

File: docs/data_management/transfer/tutorial/globus_group_management.md (hunk 3 above)
Comment (wwarriner) [reply docs/data_management/transfer/tutorial/globus_group_management.md:129]: This is a consequence of having an ordered list nested in an ordered list. I'll fix it by using an unordered list with numbers written out.

**Review by bdu-birhanu — COMMENTED** [review]
Looks good to me! Just a small typo at line 158: "Subroups" should be "Subgroups". Ok to merge on my end after that is fixed.

**Pull request merged**